### PR TITLE
Rounded Box Component

### PR DESCRIPTION
**Pull request by omartehsin1**
**Summary**
Rounded box styling was appearing in a lot of places.  A <RoundedBox> component has been created to make it easier for reuse. Contains two styles: true for roundedBox1, false for roundedBox2.

### CODE REVIEW

**Comment by aklef** [comment]
If you're already setup for this, validating the new work would be much easier for me by having before/after screenshots of the affected screens!

**Comment by timarney** [comment]
> work would be much easier for me by having before/after screenshots of the affected screens!

@aklef 

I spoke to @dsamojlenko about this - he'll give you the game plan to automate this :)

File: src/components/RoundedBox.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+};
+
+const styles = StyleSheet.create({
+  roundedBox1: {
+    marginTop: Platform.OS === 'ios' ? 5 : 20,
+    backgroundColor: 'white',
+    borderRadius: 10,
+    zIndex: -1,
+  },
+  roundedBox2: {
```
Comment (timarney) [inline src/components/RoundedBox.tsx:27]: As discussed lets pull out these "global" box styles into a new style and pass a `isFirst` which adds the marginTop etc... properties to the Box.

File: ios/CovidShield.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -571,7 +571,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = ca.gc.hcsc.canada.stopcovid;
 				PRODUCT_NAME = CovidShield;
-				PROVISIONING_PROFILE_SPECIFIER = "Stop COVID Development";
```
Comment (timarney) [inline ios/CovidShield.xcodeproj/project.pbxproj]: This file is your nemesis ;)

Comment (omartehsin1) [reply ios/CovidShield.xcodeproj/project.pbxproj]: Lol, sorry, already been reversed.

**Review by jeberhardt — APPROVED** [review]

**Comment by timarney** [comment]
Please hold from merging this until after the next release.

**Comment by timarney** [comment]
@omartehsin1 please fix up the conflicts and we'll merge this one.

**Comment by aklef** [comment]
There is now a e2e testing script using Detox that iterates through every page of the app available on the debug menu that takes screenshots as it goes. Should now be able to use that to get before/after images and then use Tim's test-galery script to generate an html page of before after...

**Pull request merged**